spyder/plugins/findinfiles/widgets/main_widget.py

```diff
@@ -171,10 +171,17 @@ def __init__(self, name=None, plugin=None, parent=None):
             self,
             items=search_text,
             adjust_to_minimum=False,
-            id_=FindInFilesWidgetToolbarItems.SearchPatternCombo
+            id_=FindInFilesWidgetToolbarItems.SearchPatternCombo,
+            items_elide_mode=Qt.ElideMiddle
         )
         self.search_text_edit.lineEdit().setPlaceholderText(
             _('Write text to search'))
+        
+        self.search_text_edit.setMinimumSize(
+            MIN_COMBOBOX_WIDTH, AppStyle.FindHeight
+        )        
+        
+        self.search_text_edit.setMaximumWidth(MAX_COMBOBOX_WIDTH)
```

Comment:
```suggestion
        # This is necessary to prevent the width of search_text_edit to control
        # the width of the pane.
        # Fixes spyder-ide/spyder#24188
        self.search_text_edit.setMaximumWidth(MAX_COMBOBOX_WIDTH)
```